app/markdowneditor.cpp

```diff
@@ -170,6 +170,11 @@ void MarkdownEditor::paintEvent(QPaintEvent *e)
     if (showHardLinebreaks) {
         drawLineEndMarker(e);
     }
+
+    // draw column ruler
+    if(rulerEnabled) {
```

Comment:
I try to follow the Qt source code style. This means a space between the `if` keyword and the parenthesis.